app/tests/integration/nimbus/pages/branches.py

```diff
@@ -0,0 +1,49 @@
+from nimbus.pages.base import Base
+
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support import expected_conditions as EC
+
+
+
+class BranchesPage(Base):
+    """Experiment Branches Page."""
+
+    _reference_branch_name_locator = (By.CSS_SELECTOR, "#referenceBranch-name")
+    _reference_branch_description_locator = (By.CSS_SELECTOR, "#referenceBranch-description")
+    _remove_branch_locator = (By.CSS_SELECTOR, ".bg-transparent")
```

Comment:
Q: Do we need to add a better classname or something to this button?

Reply:
maybe, but its the only button with that class name on the page

Reply:
Yeah good catch @pdehaan probably a good idea to give it an ID